Modules/_ctypes/_ctypes.c

```diff
@@ -5741,6 +5741,23 @@ wstring_at(const wchar_t *ptr, int size)
     return PyUnicode_FromWideChar(ptr, ssize);
 }
 
+static PyObject *
+memoryview_at(void *ptr, Py_ssize_t size, int readonly)
+{
+    Py_ssize_t ssize = size;
```

Comment:
Why this additional conversion? we are given a `ssize_t` as input so it wouldn't change anything here, unless you want that variable only for semantic purposes.

Reply:
Thanks! Yes, I left that in from `wstring_at`